DayrSuryan/DSEthiop3.xml

```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>ʾArgānona Māryām</title>
+            <title>Organ of Mary, Malkǝʾ-hymn to Mary</title>
```

Comment:
The simplest for Malke' is "Image of...", straight forward.

Reply:
Otherwise just Malke'-hymn.

Reply:
I used Malke'-hymn, because it is a specific genre, which this makes more obvious